src/styles.d.ts

```diff
@@ -45,6 +45,10 @@ export interface Theme {
   };
 }
 
+export interface Active {
+  isActive: boolean;
+}
```

Comment:
This doesn't feel general enough to live at top level like this. This type is probably an unnecessary abstraction tbh, you could just declare it inline on the button it's used in